manifests/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -215,19 +195,3 @@ spec:
       - name: config-volume
         configMap:
           name: istio{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
-  {{- if .Values.pilot.jwksResolverExtraRootCA }}
```

Comment:
no idea what this stuff does - needs a security review.

Reply:
see my comment above